src/battle_script_commands.c

```diff
@@ -11224,6 +11224,54 @@ static void Cmd_various(void)
         gBattlescriptCurrInstr = cmd->nextInstr;
         return;
     }
+    case VARIOUS_TRY_REVIVAL_BLESSING:
+    {
+        VARIOUS_ARGS(const u8 *jumpInstr);
```

Comment:
Very nit-picky, but I'd possibly call this `failInstr` both here and in `battle_script.inc`? I can't remember exactly what was common between the `jumpInstr`s but I'm guessing they're usually for jump targets where something has happened successfully (although those should be `successInstr` really...) and stuff like `goto`.